packages/editor/src/components/browser/Browser.tsx

```diff
@@ -4,15 +4,14 @@ import { Button, TabContent, TabList, TabRoot } from '../widgets';
 import { IvyIcons } from '@axonivy/editor-icons';
 import { useEditorContext } from '../../context';
 import { useState } from 'react';
-import { useAttributeBrowser } from './AttributeBrowser';
 import type { BrowserType, UseBrowserReturnValue } from './useBrowser';
-import { useFuncBrowser } from './FunctionBrowser';
-import { useTypeBrowser } from './TypeBrowser';
-import { useTableColBrowser } from './TableColBrowser';
-import { useSqlOpBrowser } from './SqlOperationBrowser';
-import { useCatPathChooserBrowser } from './CatPathChooser';
-import type { CmsOptions } from './CmsBrowser';
-import { useCmsBrowser } from './CmsBrowser';
+import { useAttributeBrowser } from './attributeBrowser/AttributeBrowser';
```

Comment:
we are already in the browser folder. Why duplicate it here?
just call them what browser is inside. `attributeBrowser` -> `attribute`.
And of course for the other too!